src/backend/marsha/core/serializers.py

```diff
@@ -18,7 +18,7 @@
 # This regex matches keys in AWS for videos or timed text tracks
 KEY_PATTERN = (
     "^(?P<resource_id>{uuid:s})/(?P<model_name>video|timedtexttrack)/(?P<object_id>{uuid:s})/"
-    "(?P<stamp>[0-9]{{10}})(_[a-z]{{2}}(_cc)?)?$".format(uuid=UUID_REGEX)
+    "(?P<stamp>[0-9]{{10}})(_[a-z]{{2}}_(cc|st|ts))?$".format(uuid=UUID_REGEX)
```

Comment:
maybe you can do something like `"|".join(m[0] for m in TimedTextTrack.MODE_CHOICES)` ? It will automatically add new timed text. I'm not sure at all about the syntax
```suggestion
    "(?P<stamp>[0-9]{{10}})(_[a-z]{{2}}_("+"|".join(m[0] for m in TimedTextTrack.MODE_CHOICES)+"))?$".format(uuid=UUID_REGEX)
```

Reply:
Great suggestion! 👍